src/gt4py/next/program_processors/runners/dace_iterator/itir_to_sdfg.py

```diff
@@ -119,11 +121,13 @@ def _make_array_shape_and_strides(
             neighbor_tables[dim.value].max_neighbors
             if dim.kind == DimensionKind.LOCAL
             # we reuse the same gt4py symbol for field size passed as scalar argument which is used in closure domain
-            else dace.symbol(f"__{name}_size_{i}", dtype)
+            else dace.symbol(connectivity_table_size_symbol(name, i), dtype)
```

Comment:
Since this is used only for the neighbor_tables, I thought that the name would be a good fit. Also, it is used in the SDFGConvertible interface, for adding the connectivity tables symbols in the fused SDFG.
If you are ok with it, I would prefer to leave it as is.

Reply:
The function `_make_array_shape_and_strides` is used for all fields, not only connectivity tables.

Reply:
You are right then. I will modify the name and ping you.

Reply:
Thanks, it looks good to me now.